app/src/main/java/fr/free/nrw/commons/mwapi/OkHttpJsonApiClient.java

```diff
@@ -58,6 +59,7 @@
     private final String commonsBaseUrl;
     private final JsonKvStore defaultKvStore;
     private Gson gson;
+    private MediaWikiApi mediaWikiApi;
```

Comment:
Don't reference `MediaWikiApi` from this class.  It already has `commonsBaseUrl` which can be used to build a new `HttpUrl` to call a commons MediaWiki API.